Classes/Eid/PageViewProxy.php

```diff
@@ -130,7 +130,9 @@ protected function handleGet(ServerRequestInterface $request): ResponseInterface
 
         // get and verify the uHash
         $uHash = (string) ($queryParams['uHash'] ?? '');
-        if (!hash_equals(GeneralUtility::hmac($url, 'PageViewProxy'), $uHash)) {
+        if ($uHash == 'View3DController') {
+            // don't verify hash
+        } else if (!hash_equals(GeneralUtility::hmac($url, 'PageViewProxy'), $uHash)) {
```

Comment:
See above: this should stay unchanged for security reasons.
```suggestion
        if (!hash_equals(GeneralUtility::hmac($url, 'PageViewProxy'), $uHash)) {
```